en/docs/learn/data-publishing-reports.md

```diff
@@ -108,15 +108,37 @@ following reports according to the OBIE templates:
 | 1 | ReportMonth | The respective table you get the month from | Get the year-month values from the table |
 | 2 | ASPSP Brand ID | N/A | A fixed value for a bank |
 | 3 | Retail/Business PSUs | N/A | The retail and business PSU details are available in the core banking system |
-| 4 | API Service | | REQUEST_TYPE from any table |
-| 5 | One-off Consents | ONE_OFF_CONSENT_ADOPTION | CONSENT_COUNT per REQUEST_TYPE |
-| 6 | Long-lived consents - Outstanding consents BoP | LONG_LIVED_CONSENT_ADOPTION | |
-| 7 | Long-lived consents - Revoked consents | LONG_LIVED_CONSENT_ADOPTION | |
-| 8 | Long-lived consents - Expired consents | LONG_LIVED_CONSENT_ADOPTION | |
-| 9 | Long-lived consents - Renewed consents | LONG_LIVED_CONSENT_ADOPTION | |
-| 10 | Long-lived consents - New consents | LONG_LIVED_CONSENT_ADOPTION | |
-| 11 | Long-lived consents - Outstanding consents EoP | LONG_LIVED_CONSENT_ADOPTION | |
-| 12 | Version | LONG_LIVED_CONSENT_ADOPTION | |
+| 4 | API Service | | `REQUEST_TYPE` from any table |
+| 5 | One-off Consents | `ONE_OFF_CONSENT_ADOPTION` | `CONSENT_COUNT` per `REQUEST_TYPE` |
+| 6 | Long-lived consents - Outstanding consents BoP | `LONG_LIVED_CONSENT_ADOPTION` | |
+| 7 | Long-lived consents - Revoked consents | `LONG_LIVED_CONSENT_ADOPTION` | |
+| 8 | Long-lived consents - Expired consents | `LONG_LIVED_CONSENT_ADOPTION` | |
+| 9 | Long-lived consents - Consents Requiring Reauthentication | `CONSENT_REQ_REAUTHENTICATION` | This is the total number of unique Long-lived consents, which required 90 days PSU re-authentication. <br/> <br/> See the note below this table to learn how to capture data. |
+| 10 | Long-lived consents - Renewed consents | `LONG_LIVED_CONSENT_ADOPTION` | |
+| 11 | Long-lived consents - New consents | `LONG_LIVED_CONSENT_ADOPTION` | |
+| 12 | Long-lived consents - Outstanding consents EoP | `LONG_LIVED_CONSENT_ADOPTION` | |
+| 13 | Long-lived consents - Total Refreshed consents | `REFRSHED_CONSENT` | This is the total number of unique Long-lived consents that have been refreshed (Re-authenticated). <br/> <br/> See the note below this table to learn how to capture data. |
+| 14 | Version | `LONG_LIVED_CONSENT_ADOPTION` | |
+
+!!! note "To capture Long-lived consents - Consents Requiring Re-authentication:"
+       1. Join `ACCOUNTS_RAW_DATA` and `API_INVOCATION_RAW_DATA` tables.
+       2. Get the total count of `CONSENT_ID` as `TOTAL_CONSENT_REQ_REAUTH`. 
+       3. Consider the following conditions:
+          - `CONSENT_TYPE` = `'long-lived'`
+          - `AUTHORISATION_STATUS` = `'Authorised'`
+          - `RE_AUTHORISATION` == `false`
+          - `EXPIRATION_DATE_TIME` > `REPORTING_DATE_TIME`
+          - `REPORTING_DATE_TIME` - 90 days> `AUTHORIZED_TIME_DATE`
+
+!!! note "To capture Long-lived consents - Total Refreshed consents:"
```

Comment:
Lets format the field name

Reply:
Fixed in fae1a3d4d37ff71f5893786f015c13654abdc2d9.